content/en/blog/2023/otel-end-user-discussions-january-2023/index.md

```diff
@@ -0,0 +1,245 @@
+---
```

Comment:
Given that this is a simple post without images, you can just rename this file:

- From `content/en/blog/2023/otel-end-user-discussions-january-2023/index.md`
- To `content/en/blog/2023/otel-end-user-discussions-january-2023.md`

Reply:
Done!